test/validation_negative.json

```diff
@@ -1,38 +1,50 @@
 {
```

Comment:
Is it worth refactoring this negative test stuff at all to check that they are failing on the expected errors? When I dug into this on monday the expired message test was erroring earlier than it should have, but since the test harness only checks for exceptions the test was passing.

Reply:
Yeah I'm not sure. Because the goal of those tests is to be used for any implementation, that would mean we'd have to specify error messages. I think, past the fact we had some issues with addresses formatting, the tests should always be correct. I double checked that current negative tests are failing for the correct reason.